### PR TITLE
Add animation for fading-in and fading-out

### LINKED ISSUE #10: Animation/Transition when search modal is closed
Currently, closing the search modal has got no animation.
It would be nice to have some (just like the opening of the modal has)

### PR DESCRIPTION
**Pull request by dojutsu-user**
Closes #10 

Demo:

Peek 2019-06-20 20-36

### CODE REVIEW

**Comment by dojutsu-user** [comment]
Closed and reopened for the travis.

File: sphinx_search/_static/js/rtd_sphinx_search.js [hunk 1]
```diff
@@ -334,17 +334,16 @@ const showSearchModal = () => {
...
-    );
-    search_outer_wrapper.classList.add("display-block");
-
-    // sets the value of the input field to empty string and focus it.
-    let search_outer_input = document.querySelector(".search__outer__input");
-    if (search_outer_input !== null) {
-        search_outer_input.value = "";
-        search_outer_input.focus();
-    }
+    $(".search__outer__wrapper").fadeIn(400, () => {
```
Comment (davidfischer) [inline sphinx_search/_static/js/rtd_sphinx_search.js]: Did this value come from anywhere? It seems slight long to me but I don't have a particular value in mind.

Comment (dojutsu-user) [reply sphinx_search/_static/js/rtd_sphinx_search.js]: It feels smooth with this.
Should I reduce it to 200 or 100?

Comment (davidfischer) [reply sphinx_search/_static/js/rtd_sphinx_search.js]: 400 seems fine. Sounds like a thing we'll get feedback on from real users.

File: sphinx_search/_static/css/rtd_sphinx_search.css [hunk 2]
```diff
@@ -186,11 +183,13 @@
 }
 
 /* Path of each search result */
+
```
Comment (davidfischer) [inline sphinx_search/_static/css/rtd_sphinx_search.css]: These spaces are a bit strange but I don't have a very strong opinion about it.

Comment (dojutsu-user) [reply sphinx_search/_static/css/rtd_sphinx_search.css]: That was just to make the css a little cleaner.

**Comment by dojutsu-user** [comment]
Merging this one.

**Pull request merged**